examples/browser-to-server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "js-libp2p-webrtc-browser-to-server",
+  "version": "1.0.0",
+  "description": "Connect a browser to a server",
+  "type": "module",
+  "scripts": {
+    "start": "vite",
+    "go-libp2p-server": "cd ../go-libp2p-server && go build && ./go-libp2p-server"
```

Comment:
does `npm` run your scripts in a new shell? As otherwise this command would change your `CWD` implicitly, which might be confusing and not expected when the cmd exits?